docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -57,25 +57,115 @@ For static onboarding, make sure that the following parameters are correctly spe
-
-  `keytool -import -file keystore/ca11/local_ca/local_ca.cer -alias gateway_ca11 -keystore keystore/ca31/localhost/localhost.truststore.p12`
-
-  `keytool -import -file keystore/ca32/local_ca/local_ca.cer -alias gateway_ca32 -keystore keystore/ca31/localhost/localhost.truststore.p12`
-
-- Import a public key certificate of the Cloud Gateway into the truststore of all domain Gateways.
-
-  Use the following example of the keytool command  when the certificate of the Cloud Gateway running on CA31 is imported into the truststore of the domain Gateways running on CA11 and CA32:
-
-  `keytool -import -file keystore/ca31/local_ca/local_ca.cer -alias gateway_ca31 -keystore keystore/ca11/localhost/localhost.truststore.p12`
-
-  `keytool -import -file keystore/ca31/local_ca/local_ca.cer -alias gateway_ca31 -keystore keystore/ca32/localhost/localhost.truststore.p12`
+## Establishing a trust relationship between Domain API ML and the Central API ML
+
+For routing to work in a MultiTenancy configuration, the Central API ML must trust the Domain API ML, and vice versa for registration.
+In other words it is necessary that the root and, if applicable, intermediate public certificates be shared between Central API ML and Domain API ML. 
+
+The following example describes this relationship. 
+
+![Trust relation diagram](./diagrams/mt-trust-relations.png)
+
+On this example diagram:
+
+Central APIML is installed on system X, Domain APIMLs are installed on system Y and Z.
+For secure communication Domain APIML 1 and 2 are using different private keys signed with different public keys (they don't trust each other).
+Central APIML should have all public keys from the certificate chain of all Domain APIMLs (DigiCert Root CA, DigiCert Root CA1, DigiCert CA) added to its truststore to be able to register them, otherwise Central APIML will not trust them.
+Central APIML is using a private key which is sign by Local CA public key for secure communication. 
+Domain APIML 1 and 2 should have Local CA public key to be able to accept the routing requests from Central APIML, otherwise Domain APIMLs will not trust it.
+On this diagram you can see all added certificates surrounded by red dashed line.
+
+The following commands are examples of establishing a trust relationship between Domain API ML and the Central API ML for both PKCS12 certificates and when using keyrings.
+
+1. Import the root and, if applicable, the intermediate public key certificate of Domain API MLs running on systems Y and Z into the truststore of the Central API ML running on system X.
+
+  - **PKCS12**
+  
+    For PKCS12 certificates, use the following example of  keytool commands:
+  
+    `keytool -import -file sysy/keystore/local_ca/local_ca.cer -alias gateway_sysy -keystore sysx/keystore/localhost/localhost.truststore.p12`
+  
+    `keytool -import -file sysz/keystore/local_ca/local_ca.cer -alias gateway_sysz -keystore sysx/keystore/localhost/localhost.truststore.p12`
```

Comment:
maybe keyring should be the first in the list, also considering it's mentioned in the intro.